smallest_int.py

```diff
@@ -0,0 +1,8 @@
+def smallest_int(arr):
+    current = arr[0]
+
+    for num in arr:
+        if num < current:
+            current = num
```

Comment:
This is a good pattern to know.  Now switch this to use [min.](https://docs.python.org/3/library/functions.html#min)

Then read over these[ built in functions.](https://docs.python.org/3/library/functions.html)